tests/e2e/assertion-issuer-field.e2e.test.ts

```
@@ -0,0 +1,306 @@
+import { describe, it, expect, beforeAll, afterAll, beforeEach } from 'vitest';
```

ESLint

[error] 'afterAll' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)